README.md

```diff
@@ -14,10 +14,10 @@ Here is the current status of Python in Azure Functions:
 
 _What are the supported Python versions?_
 
-| Azure Functions Runtime | Python 3.6 | Python 3.7 | Python 3.8 | Python 3.9 |
-|-------------------------|------------|------------|------------|------------|
-| Azure Functions 3.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
-| Azure Functions 4.0     | &#x2713;          | &#x2713;          | &#x2713;          | &#x2713;          |
+| Azure Functions Runtime | Python 3.6 | Python 3.7 | Python 3.8 | Python 3.9                                                                                                                                                                           | Python 3.10  | Python 3.11 |
+|-------------------------|------------|------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|-------------|
+| Azure Functions 3.0     | [EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)|[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)|[EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)| [EOL](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4?tabs=net6-in-proc%2Cazure-cli%2Cwindows&pivots=programming-language-python#in-this-article)| -       |-        |
```

Comment:
Please remove `en-us` part from the URL - MSFT localization kicks in and redirects you to the right place.

```suggestion
| Azure Functions 3.0     |[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)|[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)|[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)|[EOL](https://learn.microsoft.com/azure/azure-functions/migrate-version-3-version-4)| N/A | N/A |
```

Reply:
Updated as per suggestion.Thanks